assets/js/modules/analytics/datastore/accounts.js

```diff
@@ -350,18 +345,13 @@ const baseSelectors = {
 	getAccountTicketTermsOfServiceURL: createRegistrySelector( ( select ) => ( state ) => {
 		const { accountTicketID } = state;
 		const email = select( CORE_USER ).getEmail();
+		const tosURL = select( STORE_NAME ).getServiceURL( { path: `/termsofservice/${ accountTicketID }`, query: { provisioningSignup: 'false' } } );
 
-		if ( undefined === accountTicketID || ! email ) {
+		if ( undefined === accountTicketID || ! email || ! tosURL ) {
```

Comment:
We don't need to retrieve and check for the email here now since that isn't used here (now part of `getServiceURL`.

Reply:
Updated.